gpkit/model.py

```diff
@@ -176,7 +176,7 @@ def zero_lower_unbounded_variables(self):
                      if bound == "lower"}
             self.substitutions.update(zeros)
 
-    def solve(self, solver=None, verbosity=1, skipfailures=True,
+    def solve(self, solver=None, verbosity=1, skipfailures=False,
```

Comment:
I think this is minor, and I think I agree with this change, but can we just discuss what happened here with the `skipfailures` change so I understand? (any change to `solve` gets my attention)